### ISSUE TITLE
Update Words' NEWS for 0.4 release

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11196]
|[<img alt="ralphm's avatar" src="https://avatars.githubusercontent.com/u/116741?s=50" width="50" height="50">](https://github.com/ralphm)| @ralphm reported|
|-|-|
|Trac ID|trac#1714|
|Type|enhancement|
|Created|2006-05-13 19:16:42Z|

<details><summary>Searchable metadata</summary>

```
trac-id__1714 1714
type__enhancement enhancement
reporter__ralphm ralphm
priority__normal normal
milestone__Words_0_4 Words-0.4
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__words words
keywords__ 
time__1147547802000000 1147547802000000
changetime__1148257478000000 1148257478000000
version__ 
owner__ 
cc__exarkun
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="ralphm's avatar" src="https://avatars.githubusercontent.com/u/116741?s=50" width="50" height="50">](https://github.com/ralphm)|@ralphm commented|
|-|-|

Maybe the README could use some loving, too.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

Milestone Twisted-2.3 deleted

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set status to `closed`|
|-|-|

(In [16910]) News and Versioning for Twisted Words 0.4.0 release

Author: Release Automation, exarkun
Reviewer: exarkun, radix
Fixes [#11214](https://github.com/twisted/twisted/issues/11214) and [#11196](https://github.com/twisted/twisted/issues/11196)

This also removes the twisted.web.resource import from twisted.words.service, since it is
unused and creates a pointless dependency on twisted.web.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
